serpentTools/parsers/sensitivity.py

```diff
@@ -331,6 +331,10 @@ def plot(self, resp, zai=None, pert=None, mat=None, sigma=3,
         mat: None or str or list of strings
             Plot sensitivities due to these materials. Passing ``None``
             will plot against all materials.
+        mevscale: bool, optional
```

Comment:
`mevscale` is not defined in the method call and [causes the tests to fail](https://travis-ci.com/github/CORE-GATECH-GROUP/serpent-tools/jobs/336513292#L1701-L1706)